docs/understand/sizing-guide.md

```diff
@@ -298,7 +285,75 @@ The average throughput during the experiment was **7069.03 requests/second**.
 </div>
 
 <div markdown="1" id="#dynamodb-bench">
-TBD Bench
+All benchmarks below were measured using m5.xlarge instance on AWS us-east-1.
+
+### Random Writes
+
+This test generates random write requests to a given lakeFS branch.
+All the paths are pre-generated and don't overwrite each other (as overwrites are relatively rare in a Data Lake setup).
+
+**command executed:**
+
+```shell
+lakectl abuse random-write \
+    --amount 500000 \
+    --parallelism 64 \
+    lakefs://example-repo/main
+```
+
+**Note** lakeFS version <= v0.33.1 uses '@' (instead of '/') as separator between repository and branch.
+
+**Result Histogram (raw): Provisioned read capacity units - 1000
+Provisioned write capacity units - 1000**
+
+```
+Histogram (ms):
+1	0
+2	0
+5	0
+7	0
+10	0
+15	0
+25	24
+50	239852
+75	458504
+100	485225
+250	493687
+350	493872
+500	493960
+750	496239
+1000	499194
+5000	500000
+min	23
+max	4437
+total	500000
+```
+**Result Histogram (raw): Provisioned read capacity units - 500
+Provisioned write capacity units - 500**
+
+```
+Histogram (ms):
+1	0
+2	0
+5	0
+7	0
+10	0
+15	0
+25	46
+50	244800
+75	458098
+100	486082
+250	493376
+350	493647
+500	493773
+750	496227
+1000	499268
+5000	500000
+min	21
+max	4588
+total	500000
+```
+
```

Comment:
Metrics section is currently irrelevant for dynamodb and should be included only in the postgres section

Reply:
I don't understand why?

Reply:
I want to correct my statement. We no longer collect db metrics in KV

Reply:
done